src/project-3.js

```diff
@@ -5,68 +5,102 @@ const makeCat = (name, age) => {
   // add an age property to the object with the value set to the age argument
   // add a method called meow that returns the string 'Meow!'
   // return the object
+  const Cat = {
+    name,
+    age,
```

Comment:
It looks like you need to do a little work here. Remember that if you're simply passing in name and age this way, they'll need to be defined beforehand. So doing something like `name: 'Foo', age: 'Bar'` would work. Otherwise you'll need name and age declared before hand for this short-hand syntax. Also the method passed to the object can be done like so `meow: function () { //do your stuff here }` This should help.